src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -38,6 +41,8 @@ export default class EthereumBlockchainService implements BlockchainService {
       this.provider = ethers.getDefaultProvider(network);
     }
 
+    this.provider.pollingInterval = POLLING_INTERVAL
```

Comment:
Is this interval polling as in retrying to send the tx, or to poll the tx status, or both?

Reply:
It's used internally by `@ethersproject`'s Providers to control the frequency of its poll loop, which can be used for any number of registered tasks.  I think in our case however it's primarily how often the `this.provider.waitForTransaction()` call polls internally for the txn to get mined